README.md

```diff
@@ -99,10 +110,27 @@ future regressions) if you can add a small unit test which replicates the bug.
 Before submitting a change, please run tox to check that you have not
 introduced any regressions or violated any code style guidelines.
 
+### Mypy
+
+Statick Markdown uses [mypy](http://mypy-lang.org/) to check that type hints are being followed properly.
+Type hints are described in [PEP 484](https://www.python.org/dev/peps/pep-0484/) and allow for static typing in Python.
+To determine if proper types are being used in Statick Markdown the following command will show any errors, and create several
+types of reports that can be viewed with a text editor or web browser.
+
+```shell
+python3 -m pip install mypy
+mkdir report
+mypy --ignore-missing-imports --allow-untyped-decorators --strict --html-report report/ --txt-report report src
```

Comment:
```suggestion
mypy --ignore-missing-imports --strict --html-report report/ --txt-report report src
```

The `--allow-untyped-decorators` flag is not needed. It is used by Statick for the `deprecated` module.